mythril/laser/ethereum/util.py

```diff
@@ -10,46 +12,38 @@
 TT255 = 2 ** 255
 
 
-def sha3(seed):
+def sha3(seed: str) -> bytes:
     return _sha3.keccak_256(bytes(seed)).digest()
```

Comment:
i think bytes doesn't take straight string as input. as it will giveout some error with that.It seems to take some Iterable[Int] as an input.